sdks/go/model/opSpec.go

```diff
@@ -13,6 +13,7 @@ type OpSpec struct {
 //CallSpec is a spec for a node of a call graph; see https://en.wikipedia.org/wiki/Call_graph
 type CallSpec struct {
 	Container    *ContainerCallSpec    `json:"container,omitempty"`
+	Description  string                `json:"description,omitempty"`
```

Comment:
Why isn't this a `*string` reference, like `Name`?

Reply:
Just because it was `string` [on parameter already](https://github.com/opctl/opctl/blob/8a41a4b8da60b8a34ea1426e004462622c62ad53/sdks/go/model/param.go#L27) and for our use cases differentiating `nil` vs empty isn't meaningful. `Name` probably should be same but it already exists.